doc/en/components/dock-manager.md

```diff
@@ -373,6 +373,60 @@ The shortcuts are as follows:
 
 Practice all of the above mentioned actions in the sample [`demo`](dock-manager.md#$Platform$-dock-manager-example).
 
+## Styling
+
+The Dock Manager use a sahadow DOM to encapsulate his styles and behaviors. As a result, you can't simply target their internals with the usual CSS selectors. That is why we expose components `parts` that can be targeted with the `::part` CSS selector.
+
+```css
+igc-dockmanager::part(content-pane) {
+  border-radius: 10px;
+}
+```
+
+In the following example, we will demonstrate the ability of customizing the Dock Manager through some of the exposed parts that we've provided. 
```

Comment:
In the following example, we demonstrate the ability of customizing the Dock Manager through some of the CSS parts that we've exposed.